pandas/tests/io/parser/test_encoding.py

```diff
@@ -272,6 +272,26 @@ def test_chunk_splits_multibyte_char(all_parsers):
     tm.assert_frame_equal(dfr, df)
 
 
+@skip_pyarrow
+def test_readcsv_memmap_utf8(all_parsers):
+    lines = []
+    for lnum in range(0x20, 0x10080, 0x80):
```

Comment:
Maybe add a comment what these magic numbers represent or what the block is doing.

Reply:
Added explanatory variables and comment.